README.md

```diff
@@ -15,39 +15,257 @@
 ## Description
 
 This module adds a new [scheduled_task](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) provider capable of using the more modern Version 2 Windows API for task management.
+The legacy API does not receive improvements or new features, meaning that if you want to take advantage of improvements to scheduled tasks on Windows you need to use the V2 API.
 
 ## Setup
 
 ### Beginning with scheduled_task
 
-The scheduled_task module adapts the Puppet [scheduled_task](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) resource to run using a modern API. To get started, install the module and declare 'taskscheduler_api2' as the provider, for example:
+The scheduled_task module adapts the Puppet [scheduled_task](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) resource to run using a modern API.
+To get started, install the module and any existing `scheduled_task` resources will use the V2 API **by default**.
+If you want to continue using the provider for the legacy API you will _need_ to declare that in your manifests.
+For example:
 
 ~~~ puppet
 scheduled_task { 'Run Notepad':
   command  => "notepad.exe",
   ...
-  provider => 'taskscheduler_api2'
+  provider => 'win32_taskscheduler',
 }
 ~~~
 
 ## Usage
 
-See the [Puppet resource documentation](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) for more information.
+Scheduled tasks are commonly used to kick off a script either once or on a regular cadence.
+In this first example we schedule a cleanup script to run this one time.
+
+~~~ puppet
+scheduled_task { 'Disk Cleanup': # Unique name for the scheduled task
+  command   => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe',
```

Comment:
Don't want to complicate the example, but it's a little more idiomatic in Puppet to use the [system32 fact](https://puppet.com/docs/facter/3.9/core_facts.html#system32)

Some notes on this at https://puppet.com/docs/puppet/5.3/lang_windows_file_paths.html#file-system-redirection-when-running-32-bit-puppet-on-64-bit-windows